frontend/app/routes/$lang+/_protected+/alerts+/subscribe+/index.tsx

```diff
@@ -0,0 +1,162 @@
+    return json({ errors: parsedDataResult.error.format() });
+  }
+
+  const idToken: IdToken = session.get('idToken');
+  auditService.audit('update-date.subscribe-alerts', { userId: idToken.sub });
+
+  instrumentationService.countHttpStatus('alerts.subscibe', 302);
+  return redirect(getPathById('$lang+/_protected+/home', params));
+}
+
+export default function SubscribeEmailEdit() {
+  const { i18n, t } = useTranslation(handle.i18nNamespaces);
+  const { csrfToken, preferredLanguages } = useLoaderData<typeof loader>();
+  const params = useParams();
+  const fetcher = useFetcher<typeof action>();
+  const errorSummaryId = 'error-summary';
+
+  // Keys order should match the input IDs order.
+  const errorMessages = useMemo(
+    () => ({
+      email: fetcher.data?.errors.email?._errors[0],
+      'confirm-email': fetcher.data?.errors.confirmEmail?._errors[0],
+      'input-radio-preferred-language-option-0': fetcher.data?.errors.preferredLanguage?._errors[0],
+    }),
+    [fetcher.data?.errors.email?._errors, fetcher.data?.errors.confirmEmail?._errors, fetcher.data?.errors.preferredLanguage?._errors],
+  );
+
+  const errorSummaryItems = createErrorSummaryItems(errorMessages);
+
+  useEffect(() => {
+    if (hasErrors(errorMessages)) {
+      scrollAndFocusToErrorSummary(errorSummaryId);
+    }
+  }, [errorMessages]);
+
+  return (
+    <>
+      {errorSummaryItems.length > 0 && <ErrorSummary id={errorSummaryId} errors={errorSummaryItems} />}
+      <fetcher.Form className="max-w-prose" method="post" noValidate>
+        <input type="hidden" name="_csrf" value={csrfToken} />
+        <div className="mb-6 grid gap-6 md:grid-cols-2">
+          <InputField id="email" type="email" className="w-full" label={t('alerts:subscribe.email')} maxLength={100} name="email" errorMessage={errorMessages.email} autoComplete="email" required />
+          <InputField id="confirm-email" type="email" className="w-full" label={t('alerts:subscribe.confirm-email')} maxLength={100} name="confirmEmail" errorMessage={errorMessages['confirm-email']} autoComplete="email" required />
+        </div>
+        <div className="mb-8 space-y-6">
+          {preferredLanguages.length > 0 && (
+            <InputRadios
+              id="preferred-language"
+              name="preferredLanguage"
+              legend={t('alerts:subscribe.preferred-language')}
+              options={preferredLanguages.map((language) => ({
+                children: getNameByLanguage(i18n.language, language),
+                value: language.id,
+              }))}
+              errorMessage={errorMessages['input-radio-preferred-language-option-0']}
+              required
+            />
+          )}
+        </div>
+
+        <div className="flex flex-wrap items-center gap-3">
+          <ButtonLink id="back-button" routeId="$lang+/_protected+/personal-information+/index" params={params}>
+            {t('alerts:subscribe.button.back')}
+          </ButtonLink>
```

Comment:
This should bring the user back to MSCA/MSCA-D rather than the Personal Information page.

Reply:
fixed